corehq/ex-submodules/fluff/signals.py

```diff
@@ -91,7 +91,17 @@ def catch_signal(sender, **kwargs):
     engine.dispose()
 
 
-SimpleDiff = namedtuple('SimpleDiff', 'type table_name item_name')
+@attr.s
+class SimpleDiff(object):
+    type = attr.ib()
```

Comment:
nit: is it fine to use an inbuilt function name as the name here?

Reply:
it's fine as long as it's scoped to an object